editor/editor_node.cpp

```diff
@@ -6645,6 +6647,7 @@ void EditorNode::_bind_methods() {
 
 	ADD_SIGNAL(MethodInfo("request_help_search"));
 	ADD_SIGNAL(MethodInfo("script_add_function_request", PropertyInfo(Variant::OBJECT, "obj"), PropertyInfo(Variant::STRING, "function"), PropertyInfo(Variant::PACKED_STRING_ARRAY, "args")));
+	ADD_SIGNAL(MethodInfo("project_run"));
```

Comment:
I'm not sure about the grammar here, signals are usually in the past tense like `resource_saved` or `scene_changed`, unsure what to name this, but this name sounds off to me

Reply:
That's a good point, but I don't read them as such, and the specific verb here becomes ambiguous to me at least, but it's not critical :)